src/sphinx/extensions/index.rst

```diff
@@ -17,7 +17,7 @@ Official Extensions
 Third Parties
 =============
 
-* `Gradle plugin <https://github.com/alphagov/gradle-gatling-plugin>`_
+* `Gradle plugin <https://github.com/lkishalmi/gradle-gatling-plugin>`_
```

Comment:
Yep, in Barcelona so not sure what alphagov are doing with gatling these days.  @ceeaspb is it you?